libkbfs/favorites.go

```diff
@@ -340,8 +338,9 @@ func (f *Favorites) RefreshCache(ctx context.Context) {
 	}
 }
 
-// Get returns the logged-in users list of favorites. It
-// doesn't use the cache.
+// Get returns the logged-in user's list of favorites. It uses the cache.
+// TODO: decide on behavior if we've gotten an invalidation but not an updated
```

Comment:
What are the choices here?  If we've gotten an invalidation, I think the `Get` should block on fetching the favorites (which is what should be happening in this PR if I'm reading the code right.)

Reply:
> That's what we're currently doing - I was just concerned that we don't want to block on fetching the favorites if we're offline - there's a situation whereby we get invalidated and then get offline before the new favorites arrive and then can't get the favorites at all. But it's a pretty unlikely event.

In this case, the `FavoriteList` call will just fail, and the cache will remain in place.  Maybe in that case, when `req.refresh == true`, you can just set the expiration time to a few minutes in the future?  (And then get rid of this TODO.)